### PR TITLE
adding docs for content cards release

### PR DESCRIPTION
**Pull request by sbenedicadb**
# DO NOT MERGE

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/edge/adobe-journey-optimizer/public-classes/index.md [hunk 1]
```diff
@@ -25,9 +25,14 @@ This documents lists details about the public classes and enums available in Mes
 * [Class - Message](./message.md)
 * [Enum - MessagingEdgeEventType](./messaging-edge-event-type.md)
 
-## Code-based experiences
+## Proposition-based messaging<br /> \(Code-based experiences & Content Cards)
```
Comment (calebk1) [inline src/pages/edge/adobe-journey-optimizer/public-classes/index.md:28]: Why do we have the `<br>` tag here?

**Review by calebk1 — COMMENTED** [review]
Content-wise looks mostly good!

Quick question though - I've seen both "nil" and "null" used for the returns. Are these the same? If so, I'd suggest being consistent between them. If they're not the same, we should try to show what the difference between the two of them are, since generally they're functionally the same idea.

File: src/pages/edge/adobe-journey-optimizer/public-classes/content-card-schema-data.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+```java
+PropositionItem propositionItem;
+ContentCardSchemaData contentCardSchemaData = propositionItem.getContentCardSchemaData();
+if (contentCardSchemaData != null) {
+    ContentCard contentCard = contentCardSchemaData.getContentCard();
+    if (contentCard != null) {
+        // do something with the ContentCard object
+    }
+}
+```
```
[the remark this answers is not in the document]
Comment (sbenedicadb) [reply src/pages/edge/adobe-journey-optimizer/public-classes/content-card-schema-data.md:159]: this one is the java example for the android sdk - the one above is the swift example for the ios sdk.

answering your other question about `nil` vs `null` - the former is swift, the latter is java

File: src/pages/edge/adobe-journey-optimizer/public-classes/index.md (hunk 1 above)
Comment (sbenedicadb) [reply src/pages/edge/adobe-journey-optimizer/public-classes/index.md:28]: stylistic reasons.  all the other headings on this page are short, stacking this one made it look less awkward to me :)

File: src/pages/edge/adobe-journey-optimizer/public-classes/content-card.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+- _interaction_ - a custom `String` value to be recorded in the interaction
+- _eventType_ - the [`MessagingEdgeEventType`](./messaging-edge-event-type.md) to be used for the ensuing Edge Event
+
+#### Example
+
+```java
+ContentCard contentCard;
+
+// tracking a display
+contentCardSchemaData.track(null, MessagingEdgeEventType.display);
```
Comment (rymorale) [inline src/pages/edge/adobe-journey-optimizer/public-classes/content-card.md:136]: should these examples should show contentCard.track() being called instead?

Comment (sbenedicadb) [reply src/pages/edge/adobe-journey-optimizer/public-classes/content-card.md:136]: yes!

**Pull request merged**